templates/observability/what-is-observability.html

```diff
@@ -63,7 +63,6 @@ <h3 class="p-heading--4">Metrics</h3>
         <div class="col-5 u-align--center u-hide--small u-vertically-center">
             {{ image (
                 url="https://assets.ubuntu.com/v1/df2f25e5-metrics.svg",
-                alt="Metrics",
```

Comment:
You need to leave the alt attribute to be valid mark up but the value should be an empty string.

Reply:
Sounds to me like a one-liner fix in the implementation of the tag, but in the interest of getting the update out, sure